tables/gene_classification.tsv

```diff
@@ -0,0 +1,46 @@
+gene_type	gene_class	hg19	mm9
```

Comment:
This looks fine, as long as there's documentation about the file / its columns somewhere.